tools-local/tasks/mobile.tasks/AndroidAppBuilder/AndroidAppBuilder.cs

```diff
@@ -60,7 +66,7 @@ public override bool Execute()
         apkBuilder.BuildApiLevel = BuildApiLevel;
         apkBuilder.BuildToolsVersion = BuildToolsVersion;
         apkBuilder.StripDebugSymbols = StripDebugSymbols;
-        (ApkBundlePath, ApkPackageId) = apkBuilder.BuildApk(SourceDir, abi, MainLibraryFileName, MonoRuntimeHeaders);
+        (ApkBundlePath, ApkPackageId) = apkBuilder.BuildApk(SourceDir, abi, MainLibraryFileName, MonoRuntimeHeaders, NativeMainSource);
```

Comment:
can you make it a property rather than ctor argument? (since it's very optional)

Reply:
Done :)